services/ui-src/src/components/alerts/Alert.tsx

```diff
@@ -16,23 +16,23 @@ import { parseCustomHtml } from "utils";
 // assets
 import alertIcon from "assets/icons/icon_info_circle.png";
 import warningIcon from "assets/icons/icon_warning.png";
-import errorIcon from "assets/icons/icon_error_circle.png";
+import errorIcon from "assets/icons/icon_alert_circle.png";
 
 export const Alert = ({
   status,
   title,
   description,
   link,
   showIcon = true,
-  ...props
+  sxOverride,
+  className,
 }: Props) => {
   return (
     <AlertRoot
       status={status}
       variant="left-accent"
-      sx={sx.root}
-      className={status}
-      {...props}
+      sx={{ ...sx.root, ...sxOverride?.root }}
```

Comment:
nit: I think all our other style overrides are just `...sxOverride`. can we stick with that, and reference the `root` object in the invoking component?
i.e.
```js
// component invoking
<Alert sxOverride={sx.root) />

...

<AlertRoot sx={{ ...sx.root, ...sxOverride }} />